### ISSUE TITLE
Create dataset loader for SIB-200

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#266]
Dataloader name: `sib200/sib200.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?sib200

|     Dataset| sib200 |
|-------------|---|
| Description | SIB-200 is a topic classification data set covering 200 languages and derived from Flores-200 machine translation corpus. The English subset of Flores-200 was annotated with topic labels and those labels were projected onto the parallel instances belonging to the remaining 203 languages covered in the corpus. In initial baseline experiments on SIB-200, languages unseen during pre-training of multilingual LMs, under-represented language families, and languages from regions of Africa, Americas, Oceania, and SEA often had the lowest performance. Note that, after annotation, topic categories with less than 80 sentences were removed from the final classification dataset. |
| Subsets | ace, ban, bjn, bug, ceb, ilo, ind, jav, kac, khm, lao, lus, min, mya, pag, shn, sun, tgl, tha, vie, war, zsm |
| Languages | ace, ban, bjn, bug, ceb, ilo, ind, jav, kac, khm, lao, lus, min, mya, pag, shn, sun, tgl, tha, vie, war, zsm |
| Tasks | Text Classification |
| License | Apache license 2.0 (apache-2.0) |
| Homepage | https://github.com/dadelani/sib-200 |
| HF URL | - |
| Paper URL | https://arxiv.org/pdf/2309.07445v1.pdf |

**Comment by muhsatrio** [comment]
#self-assign

**Comment by sabilmakbar** [comment]
Hi @muhsatrio, may we know the update on this dataloader issue? It's been 2 weeks since the last poke from the SEACrowd stale-checker, and we might consider unassigning if there's no progress update in the next 24 hours.

**Comment by tellarin** [comment]
@SamuelCahyawijaya I'd like to take on this loader, if the effort is staled.

**Comment by tellarin** [comment]
Can you also update the dataset card to list the HF repo? https://huggingface.co/datasets/Davlan/sib200

**Comment by holylovenia** [comment]
@tellarin Thanks for the info. Added in the datasheet. Feel free to `#self-assign` on this dataloader issue.

@muhsatrio I'm removing your assignment due to no response. 🙏
